ansible_collections/arista/avd/molecule/eos_designs-twodc-5stage-clos/molecule.yml

```diff
@@ -42,14 +42,3 @@ provisioner:
     - --inventory=inventory/hosts.yml
```

Comment:
While we are on this is it not redundant the `hosts` / `ansible_args` ?

Reply:
I don't know.

Reply:
fixed